flexget/plugins/notifiers/microsoftteams.py

```diff
@@ -0,0 +1,58 @@
+from builtins import *
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginWarning
+from requests.exceptions import RequestException
+from flexget.utils.requests import Session as RequestSession
+
+requests = RequestSession(max_retries=3)
+
+__name__ = 'microsoftteams'
+
+log = logging.getLogger(__name__)
+
+class TeamsNotifier(object):
+    """
+    Example:
+
+      teams:
+        web_hook_url: <string>
+        [message: <string>]
+        [title: <string>]
+        [themecolor: <string>]
+    """
+    schema = {
+        'type': 'object',
+        'properties': {
+            'web_hook_url': {'type': 'string'},
+            'message': {'type': 'string'},
```

Comment:
`message` should not be passed as part of the schema. It's passed via the notifiers interface directly